### PR TITLE
[1/2] Updated dependencies

### PR DESCRIPTION
**Pull request by rossjones**
Some (lots) of the dependencies are rather old and need updating.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
     "grunt-contrib-requirejs": "1.0.0",
     "grunt-contrib-watch": "1.0.0",
     "grunt-digest": "0.1.2",
-    "grunt-jasmine-node-new": "0.3.2",
+    "grunt-jasmine-node-new": "git+https://github.com/rossjones/grunt-jasmine-node",
```
Comment (pcraig3) [inline package.json]: 🙈

Comment (rossjones) [reply package.json]: I know, right. A new unmaintained fork of an unmaintained fork of an unmaintained project. We should stop using it I think.

**Comment by pcraig3** [comment]
Having another pop at this. Looks like a good place to start for [this ticket](https://trello.com/c/rq5K3WUR/119-fix-document-security-issues-in-spotlight).

**Comment by pcraig3** [comment]
Okay, I think this is good to go. Tests all pass and the app looks fine when run locally.

I've written up a little document separately that catalogues the packages we can't upgrade without some dev work and which ones we're worried about.

I've also opened a separate pull request that removes cheapseats, which we could merge or not merge as needed.

**Pull request merged**